### PR TITLE
Add Private Aggregation API testing

### PR DESCRIPTION
**Pull request by kevinkiklee**
This PR adds Private Aggregation API (PAA) testing.

* Related DCC PR: https://github.com/GoogleChrome/developer.chrome.com/pull/5304
* Related Shared Storage Demo (SSD) PR: https://github.com/GoogleChromeLabs/shared-storage-demo/pull/17

---

[Private Aggregation API](https://developer.chrome.com/docs/privacy-sandbox/private-aggregation/) allows users to report cross-site data in [Shared Storage](https://developer.chrome.com/en/docs/privacy-sandbox/shared-storage/).  

This test adds a Private Aggregation API call that reports randomly generated key/value pairs.  The reports are not collected.  

The PAA test script is injected as a third-party script.  The 3p script contains its own 3p origin trial token.

---

To test, visit the staging link (TBD), then open up `chrome://private-aggregation-internals` page to check the reported values.

### CODE REVIEW

**Comment by matthiasrohmer** [comment]
@kevinkiklee is waiting on launch approval.

**Review by devnook — APPROVED** [review]

**Pull request merged**
